mfa/about.md

```diff
@@ -0,0 +1,20 @@
+# Bandwidth MFA API
+
+## Base API URL
+`https://mfa.bandwidth.com/api/v1/accounts/{accountId}`
+
+## REST API Reference Index
+
+| Resource                                     | Description                                                           |
+|:---------------------------------------------|:----------------------------------------------------------------------|
+| [`/code/voice`](methods/code/voice.md)           | Authenticate via voice |
+| [`/code/messaging`](methods/code/messaging.md) | Authenticate via messaging |
+| [`/code/verify`](methods/code/verify.md)           | Verify an authentication request |
+                                                        |
+## Error Codes
+| Type                         | Description                                                           |
+|:-----------------------------|:----------------------------------------------------------------------|
+| [Rate Limits](rateLimits.md) | Every endpoint is rate limited                                        |
+| [HTTP Errors](errors.md)     | Learn about the different errors that you may encounter using the API |
+
+---
```

Comment:
Include a tutorial or guide on what the steps to use the service should be.

Guide can begin from "Your account is configured configured correct for use with MFA services" 

Will be relatively simple guide, step 1 use the MFA service to send a code... step 2 get the input from your end-user, step 3 validate the code with our service.

Include notes what to do when an error is hit and include information how we generate codes (TOTP) and that if a new code is requested quickly enough, they will get the same code.